samples/quickstart/create-s3-bucket.sh

```diff
@@ -0,0 +1,43 @@
+#!/bin/bash
+# Copyright 2022.
+# 
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+# 
+#     http://www.apache.org/licenses/LICENSE-2.0
+# 
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+if [ "${BUCKET_NAME}" == "" ]; then
+  printf "\n**WARNING** No BUCKET_NAME found, export it and try again\n"
```

Comment:
Since your reading the value in two lines, maybe . . .`found, export it to avoid manual entery\n`  instead

Reply:
makes sense, fixed here and in the other scripts